### PR TITLE
Created tables using diagram

### PR DESCRIPTION
**Pull request by sebalp100**
# Created tables using a diagram

For this project, we created different tables using the diagram from the requirements. We implemented the following:

- Patients table
- Medical histories table referencing patients' table
- Invoices table referencing medical histories table
- Treatments table
- Invoice items table referencing invoices and treatments table
- Join table between medical histories and treatments

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+CREATE TABLE invoice_items (
+    id SERIAL PRIMARY KEY,
+    unit_price DECIMAL,
+    quantity INTEGER,
+    total_price DECIMAL,
+    invoice_id INTEGER REFERENCES invoices(id) ON DELETE CASCADE,
+    treatment_id INTEGER REFERENCES treatments(id) ON DELETE CASCADE,
+    UNIQUE (invoice_id, treatment_id)
+);
+
```
Comment (Abenezer-Tilahun) [inline schema_based_on_diagram.sql]: - [ ] As stated in the project requirement Kindly I recommend you to  add the FK indexes for all FOREIGN KEYS.
 
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @sebalp100  @osbentech     , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git-Flow.
✔️ Good commit messages
✔️ Professional README file.

almost

## Required Changes ♻️

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (sebalp100) [reply schema_based_on_diagram.sql]: @Abenezer-Tilahun Hi, I'm not sure if I understand this requirement. Are we supposed to create indexes for the Fk ? For example, "CREATE INDEX idx_patient_id ON medical_histories (patient_id);" etc

Comment (Abenezer-Tilahun) [reply schema_based_on_diagram.sql]: @sebalp100 Yeah Just Like the example you mentioned,

**Comment by sebalp100** [comment]
FK indexes added 👍

**Review by victorokoroji — APPROVED** [review]
Hi @sebalp100, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**